src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/AnalysisController.java

```diff
@@ -3,7 +3,11 @@
 import java.security.Principal;
 import java.util.*;
```

Comment:
There is a `PREVIEWS` map at the top that is never used, can you remove it here please.

Reply:
I'll remove it in the cleanup branch that I have going